upsilonconf/io.py

```diff
@@ -69,7 +69,7 @@ def save_json(
 def load_yaml(path: Path) -> Mapping[str, Any]:
     """
     Read config from a YAML file.
-
+key_modifiers
```

Comment:
is this a typo of some sort?

Reply:
Oops, a spurious paste, from the time, when my computer decided to slow down and not acting promptly on my commands. I must have missed that. Sorry.

I've removed it already.

Reply:
Done in https://github.com/hoedt/upsilonconf/pull/3/commits/f8e4b921437c4a72ae1b230d826a37c79e28f2e5.